### ISSUE TITLE
XCode 10 GM and Swift 4.2 fail to build

### ISSUE THREAD

**Issue by jlaws** [muukii/NextGrowingTextView#71]
```
/Users/joelaws/Workspace/kratos/Pods/NextGrowingTextView/NextGrowingTextView/NextGrowingInternalTextView.swift:38:157: 'UITextViewTextDidChange' has been renamed to 'UITextView.textDidChangeNotification'
/Users/joelaws/Workspace/kratos/Pods/NextGrowingTextView/NextGrowingTextView/NextGrowingInternalTextView.swift:38:157: 'UITextViewTextDidChange' was obsoleted in Swift 4.2 (UIKit.NSNotification.Name)
```

**Comment by muukii** [comment]
@jlaws 
Thank you
I've released 1.4.0 for Swift4.2!
We can use this in Xcode10.
Please check it.

**Comment by jlaws** [comment]
Works great, thank you!
